internal/serve/graphql/dataloaders/transaction_loaders.go

```diff
@@ -17,31 +17,6 @@ type TransactionColumnsKey struct {
 	Columns       string
 }
 
-// TransactionsByAccountLoader creates a dataloader for fetching transactions by account address
-// This prevents N+1 queries when multiple accounts request their transactions
-// The loader batches multiple account addresses into a single database query
-func TransactionsByAccountLoader(models *data.Models) *dataloadgen.Loader[TransactionColumnsKey, []*types.Transaction] {
```

Comment:
Why are these removed? Does this mean we're susceptible to the N+1 problem for transactions-for-account queries? Same for the `OperationsByAccountLoader`.